### ISSUE TITLE
Details about the paper

### ISSUE THREAD

**Issue by Ayanami2019** [diviswen/PMP-Net#2]
You said in section 3.4 "In experiments, we follow the simplified algorithm in [48] to estimate an approximation of φ.", but you did not use EMD as the loss function, but used PMD to minimize the total moving distance. I don't quite understand the role of this bijective function φ, I hope you can explain it, thanks.

**Comment by diviswen** [comment]
Actually this description may be a little bit redundant. We use to apply both EMD loss (follow [48] to calculate) and CD loss, but found one of these two losses would be enough to train the model. Therefore, we finally only preserve CD for its better computational efficiency (see Eq. 10).
As for PMD and EMD, these two serves different purpose. EMD (we use CD to replace this loss) aims to regularize predicted shape close to the ground truth shape, and PMD loss regularizes the model to move points like an earth mover (which means moving points according to bijection φ, and this φ serves as an optimal target) which is define by EMD.

**Comment by Ayanami2019** [comment]
I understand what you mean, I didn’t read it carefully before, thank you for your answer
